src/Microsoft.ML.Transforms/Text/LdaTransform.cs

```diff
@@ -0,0 +1,976 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Float = System.Single;
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.CommandLine;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.EntryPoints;
+using Microsoft.ML.Runtime.Internal.Internallearn;
+using Microsoft.ML.Runtime.Internal.Utilities;
+using Microsoft.ML.Runtime.Model;
+using Microsoft.ML.Runtime.TextAnalytics;
+
+[assembly: LoadableClass(typeof(LdaTransform), typeof(LdaTransform.Arguments), typeof(SignatureDataTransform),
+    LdaTransform.UserName, LdaTransform.LoaderSignature, LdaTransform.ShortName, DocName = "transform/LdaTransform.md")]
+
+[assembly: LoadableClass(typeof(LdaTransform), null, typeof(SignatureLoadDataTransform),
+    LdaTransform.UserName, LdaTransform.LoaderSignature)]
+
+/*
+ * REVIEW: So this would be my starting point. Much of the plumbing is already
+ * done, except for the parts indicated by "REVIEW taifengw(tfinley):" in comments. I have not
```

Comment:
>REVIEW taifengw(tfinley) [](start = 44, length = 24)

weird...

Reply:
Uh right. Please just remove this comment. It was back when Taifeng was writing the transform, I furnished him with a "stub" of a transform implementation, with comments on what he would need to write where. This is no longer relevant.

---
In reply to: [197308219](https://github.com/dotnet/machinelearning/pull/377#discussion_r197308219) [](ancestors = 197308219)